src/App.tsx

```diff
@@ -1,11 +1,11 @@
-import MainPage from './components/MainPage'
 import { Routes, Route } from 'react-router-dom'
 import {
   AddFeedbackPage,
   EditFeedbackPage,
   FeedbackPage,
   RoadmapPage,
 } from './components'
```

Comment:
Here now you will import these components from `/pages` folder. Create `index.ts` file in `/pages` directory and re-export everything you need from there